src/components/Carousel/Carousel.js

```diff
@@ -111,18 +132,41 @@ class Carousel extends Component {
     });
   }
 
+  renderChildren() {
+    const { children, items } = this.props;
+    const { widthOfSlide } = this.state;
+    const style = { maxWidth: `${widthOfSlide}px` };
+    const listOfChildren = [...children];
+
+    return listOfChildren.map((item, i) => (
+      <div
+        key={items[i].id}
```

Comment:
key={item.id}
you have access to items[i] from argument as just item

Reply:
this way it gives `Warning: Each child in a list should have a unique "key" prop.` in console.

that`s only way I found yesterday to handle this warning :(

Reply:
done)

Reply:
done